beacon-chain/db/iface/interface.go

```diff
@@ -103,7 +103,7 @@ type NoHeadAccessDatabase interface {
 	// Powchain operations.
 	SavePowchainData(ctx context.Context, data *db.ETH1ChainData) error
 	// Slasher operations.
-	SaveLatestEpochAttestedForValidators(
```

Comment:
It's not the LatestEpochAttested, but rather the the last "current epoch" we have seen for validator indices in a chunk